### ISSUE TITLE
WCAG 1.4.11 – Collapse/Expand toggle arrow has insufficient contrast (2.7:1)

### ISSUE THREAD

**Issue by alunaticity** [hypothesis/product-backlog#1659]
The contrast ratio for the collapse/expand arrow (used to toggle visibility of content sections) falls just below the WCAG minimum requirement of 3:1. Measured ratio was 2.7:1.

**Comment by robertknight** [comment]
Can someone add a screenshot here to show which collapse/expand arrows this issue refers to? Any idea @acelaya or @mkdir-washington-edu ?

**Comment by acelaya** [comment]
I don't know what toggle arrow we are referring to here.

**Comment by mkdir-washington-edu** [comment]
We're doing this in response to our "partially supports" rating for [WCAG 1.4.11 Non-text Contrast ](https://www.w3.org/TR/WCAG21/#non-text-contrast)

The feedback we got on that item was:
> The collapse/expand toggle arrow is slightly below the 3:1 threshold (2.7:1)

I believe this is the arrows that make up the annotation threads in the client. If these arrows don't seem to match the feedback plesae let me know and I'll request the VPAT reviewer revisit their notes.

**Comment by acelaya** [comment]
I can confirm the "toggle arrow" we are referring to here is the chevron used to collapse/expand annotation replies:

<img width="243" height="111" alt="Image" src="https://github.com/user-attachments/assets/b24e622d-8325-4974-a19f-6d063c98c5ef" />

This uses the `#9c9c9c` color, which has a 2.75 contrast with a white background.

Our next shade of grey, `#737373` has a contrast of 4.75, so it should be good enough:

<img width="243" height="111" alt="Image" src="https://github.com/user-attachments/assets/3d39953d-f7e9-4577-ae97-d47720cf8a20" />

EDIT: We have applied the color change above.
